concepts/observability-strategy/configurable-tracing/pocs/w3c-tracecontext/README.md

```diff
@@ -0,0 +1,34 @@
+## Goal
+
+Prove that istio can be switched to w3c-tracecontext protocol and verify that Kyma components will support it native
+
+## Strategy
+
+At the moment, the only solution to use w3c-tracecontext already is via an openCensus tracer for istio. As Jaeger does not support the openCensus protocol, an otel-collector needs to be introduced additionally which does the conversion to the zipkin/jaeger protocol. As the goal anyway is to introduce an otel-collector, that approach sournds feasible as the central otel-collector could support openCensus temporarily as additional receiver.
+
+## Setup
+
+The activation of the openCensusAgent as tracer via the [extensionProvider](https://preliminary.istio.io/latest/docs/tasks/observability/distributed-tracing/opencensusagent/) concept was not working proper yet. Here, the sampling rate was not taken into account proper with strange effects. The istio-ingressgateway was not reporting spans, the downstream services received trace headers without sampling decision and still were reporting spans.
+
+So the PoC used the traditional approach via the meshConfig and an hardcoded proxy setting. An sample application based on Serverless and Eventing was deployed.
+
+1. Have an OSS Kyma 2.6.1 installed
+2. Deploy the otel-collector `kubectl -n kyma-system apply -f otel-collector.yaml`
+3. Edit kyma/resources/istio/values.yaml and add the values taken from [istio-changes.yaml](./istio-changes.yaml)
+4. Update istio: `kyma deploy -s local --component istio`
+5. Verify changes are applied: `kubectl -n istio-system get istiooperators.install.istio.io -oyaml`
+6. Deploy sample app
+```
```

Comment:
indent the whole code block with 3 blanks so it's rendered correctly under step 6.